pkg/policyfilter/map.go

```diff
@@ -78,6 +91,75 @@ func (m PfMap) release() error {
 	return nil
 }
 
+// release closes the policy filter bpf map and remove (unpin) the bpffs file
+func (m PfMap) release() error {
+	if err := releaseMap(m.dir); err != nil {
+		return err
+	}
+
+	if err := releaseMap(m.rev); err != nil {
+		return err
+	}
```

Comment:
not sure this is actually relevant: should you try to release both maps here and maybe join error? So that you are still trying to release as maximum as possible in the case the first one fails?

Reply:
Yes, this makes sense. I have updated that to use `errors.Join`.